tests/unit/test_depend_utils.py

```diff
@@ -58,6 +59,7 @@ def test_ctypes_LibraryLoader_LoadLibrary(monkeypatch, classname):
     assert res == set(['somelib.xxx'])
 
 
+@xfail(is_macos_11, reason='System libraries are hidden on macOS 11.')
```

Comment:
Maybe should we just `skip`?

Reply:
Good point. I'll push a [skip ci] commit to change that once the current round of tests finishes.